components/core/Services/MaskInputConverterFactory.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace AntDesign;
+
+public static class MaskInputConverterFactory
+{
+    public static MaskInputConverter CreateForDateTime()
```

Comment:
I think this factory can be abstracted into a converter interface, so that users can implement their own converter.

Reply:
Ant this implement is good to name `DateTimeInputMaskConverter`